presto-bytecode/src/main/java/com/facebook/presto/bytecode/ClassDefinition.java

```diff
@@ -121,10 +122,16 @@ public String getSource()
         return ImmutableList.copyOf(methods);
     }
 
+    public boolean isInterface()
+    {
+        return access.contains(INTERFACE);
+    }
+
     public void visit(ClassVisitor visitor)
     {
         // Generic signature if super class or any interface is generic
         String signature = null;
+        boolean isInterface = access.contains(INTERFACE);
```

Comment:
Why not use `isInterface()` and remove this variable?

Also, the normal naming convention for booleans would be `interface`. The `is` prefix is for getters (instead of `get`).